src/ExtendedTableWidget.cpp

```diff
@@ -314,35 +322,42 @@ void ExtendedTableWidget::copy(const bool withHeaders)
     const QString fieldSepText = "\t";
     const QString rowSepText = "\r\n";
 
+    QString sqlInsertStatement = QString("INSERT INTO %1 ( '").arg(sqlb::escapeIdentifier(m->currentTableName().toString()));
     // Table headers
-    if (withHeaders) {
+    if (withHeaders || inSQL) {
         htmlResult.append("<tr><th>");
         int firstColumn = indices.front().column();
         for(int i = firstColumn; i <= indices.back().column(); i++) {
             QByteArray headerText = model()->headerData(i, Qt::Horizontal, Qt::DisplayRole).toByteArray();
             if (i != firstColumn) {
                 result.append(fieldSepText);
                 htmlResult.append("</th><th>");
+                sqlInsertStatement.append("', '");
             }
 
             result.append(escapeCopiedData(headerText));
             htmlResult.append(headerText);
+            sqlInsertStatement.append(escapeCopiedData(headerText));
```

Comment:
`sqlb::escapeIdentifier` should be used here too for column names instead of `escapeCopiedData`, and the ' single quotes' removed.